### DISCUSSION TITLE
MEEP installation error

### DISCUSSION THREAD

**Discussion by tshwang0928** [NanoComp/meep#2061] [Q&A] [answered]
[Environment]
- Mac OS Monterey Ver. 12.3.1
- MacBook Pro(Apple M1)

Hi,

I've tried to install the MEEP using "_conda create -n mp -c conda-forge pymeep pymeep-extras_" after I installed miniconda. I got the error below.

>> PackagesNotFoundError: The following packages are not available from current channels:
   - pymeep-extras
   - pymeep

>> Current channels:
  - https://conda.anaconda.org/conda-forge/osx-arm64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/osx-arm64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/osx-arm64
  - https://repo.anaconda.com/pkgs/r/noarch

Even though I installed MEEP by building from source files with extra libraries, there was an error (cannot import meep as mp) when I ran the tutorial-basic example. 

Can I get some advice on the installation?

**Comment by oskooi** [comment, marked as answer]
The [PyMeep Conda package for macOS](https://anaconda.org/conda-forge/pymeep) is only for osx-64. Apple M1 is not yet supported, unfortunately. See also #1853.

**Comment by dalarev** [comment]
FYI I had a lot of luck using the script in [this](https://github.com/NanoComp/meep/issues/1853#issuecomment-1173570229) comment in the thread linked above.

User Ds1312 (can't @ him for some reason) was nice enough to automate the process via a script! Just be sure to choose your preferred Conda environment name before running the script. I'm having a hard time (read: failing) to change that now.

edit: aaaand I broke it trying to install/uninstall Jupyter via conda.
